Description: Fix getsockopt() on hppa
 PA-RISC overrides getsockopt() because it requires the last parameter
 to be passed on the stack to the function, and that needs to be
 translated to the kernel system call - and a recent renaming of
 getsockopt.S to __getsockopt.S (including a change to a weak symbol)
 was not done for PA-RISC, causing getsockopt() to always fail with
 "Bad Address".
Author: Christian Seiler <christian@iwakd.de>
Last-Update: 2017-01-25

--- /dev/null
+++ b/parisc/__getsockopt.S
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall5_weak(getsockopt, getsockopt, __libc_getsockopt);
--- a/parisc/getsockopt.S
+++ /dev/null
@@ -1,3 +0,0 @@
-#include "syscalls.h"
-
-syscall5(getsockopt, getsockopt);
